File: core/tests/fixtures/plugins/capacitor-coverage-edge-cases/fixture-1-bridge-contract/android/BridgePlugin.kt
@CapacitorPlugin(name = "Bridge")
class BridgePlugin : Plugin() {
  @PluginMethod
  fun liveMethod(call: PluginCall) { call.resolve() }

  @PluginMethod
  fun phantomBridge(call: PluginCall) { call.resolve() }

  @PluginMethod
  fun orphanNative(call: PluginCall) { call.resolve() }
}

File: core/tests/fixtures/plugins/capacitor-coverage-edge-cases/fixture-1-bridge-contract/ios/BridgePlugin.swift
@objc(BridgePlugin)
public class BridgePlugin: CAPPlugin {
  @objc func liveMethod(_ call: CAPPluginCall) { call.resolve() }
  @objc func phantomBridge(_ call: CAPPluginCall) { call.resolve() }
  @objc func orphanNative(_ call: CAPPluginCall) { call.resolve() }
}
